lib/completion.ts

```diff
@@ -45,6 +51,10 @@ export class Completion implements CompletionInstance {
       (this.shim.getEnv('SHELL')?.includes('zsh') ||
         this.shim.getEnv('ZSH_NAME')?.includes('zsh')) ??
       false;
+    this.fishShell = this.shim.getEnv('SHELL')?.includes('fish') ?? false;
+    this.bashShell =
```

Comment:
I get the intent, but the combination of the name `bashShell` and the use in the later login block are confusing. I'll comment more there.